sdk/python/tests/compiler/testdata/basic.yaml

```diff
@@ -14,6 +14,11 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Workflow
 metadata:
+  annotations:
+    kubeflow.org/pipelines/pipeline_spec: '{"name": "Save Most Frequent", "description":
+      "Get Most Frequent Word and Save to GCS", "inputs": [{"name": "message", "description":
+      "", "type": "", "default": null}, {"name": "outputpath", "description": "",
+      "type": "", "default": null}]}'
```

Comment:
I believe these are not the complete list of input params for pipeline, but more like metadata which we can with UI et all?

Reply:
>these are not the complete list of input params for pipeline

This list should have all inputs.

It should be the same as component.yaml without the implementation section.